test/system/irohad_test.cpp

```diff
@@ -36,29 +38,51 @@ using namespace std::chrono_literals;
 using namespace common_constants;
 using iroha::operator|;
 
+static logger::LoggerManagerTreePtr getIrohadTestLoggerManager() {
+  static logger::LoggerManagerTreePtr irohad_test_logger_manager;
+  if (!irohad_test_logger_manager) {
+    irohad_test_logger_manager =
+        std::make_shared<logger::LoggerManagerTree>(logger::LoggerConfig{
+            logger::LogLevel::kInfo, logger::kDefaultLogPatterns});
+  }
+  return irohad_test_logger_manager->getChild("IrohadTest");
+}
+
 class IrohadTest : public AcceptanceFixture {
  public:
-  IrohadTest() : kAddress("127.0.0.1"), kPort(50051) {}
+  IrohadTest()
+      : kAddress("127.0.0.1"),
+        kPort(50051),
+        test_data_path_(boost::filesystem::path(PATHTESTDATA)),
+        keys_manager_(
+            kAdminId,
+            test_data_path_,
+            getIrohadTestLoggerManager()->getChild("KeysManager")->getLogger()),
+        log_(getIrohadTestLoggerManager()->getLogger()) {}
 
   void SetUp() override {
     setPaths();
-    auto config = parse_iroha_config(path_config_.string());
+
+    rapidjson::Document doc;
+    std::ifstream ifs_iroha(path_config_.string());
+    rapidjson::IStreamWrapper isw(ifs_iroha);
```

Comment:
This name is OK. Since now I would not expect rejects for that kind of names for "very" local variables.